tests/Meilisearch.Tests/SearchTests.cs

```diff
@@ -32,6 +33,33 @@ public async Task InitializeAsync()
 
         public Task DisposeAsync() => Task.CompletedTask;
 
+        [Fact]
+        public async Task TestJsonConverter()
```

Comment:
Why is this method asynchronous if there is no `await` inside?

Reply:
I guess it's just a habit of making everything async 😄